ui/apps/platform/src/Containers/Collections/CollectionsTablePage.tsx

```diff
@@ -52,15 +52,15 @@ function CollectionsTablePage({ hasWriteAccessForCollections }: CollectionsTable
     );
     const {
         data: listData,
-        loading: listLoading,
+        isLoading: listLoading,
```

Comment:
@dvail Although one side of my brain liked the parallel naming at the time, what do you think about pro and con of a follow up contribution to rename the two quartets similar to current patterns in vulnerabilities?

Reply:
You mean instead of destructuring immediately, refer to the return value as e.g. `listQueryResult` and access properties off that object?

Reply:
Only if that is the pattern. What I was thinking was `listLoading` and `countLoading` become more like `isLoadingList` and `isLoadingCount` plus any other adjustments to current pattern for the quartet of properties.

Reply:
The `countLoading` got my attention because I read as verb and then backed up to reread as noun.